eoscompanion/content_adjusters.py

```diff
@@ -26,7 +26,7 @@
     rewrite_ekn_url
 )
 
-_RE_EKN_URL_CAPTURE = re.compile(r'"ekn\:\/\/\/(?P<id>[a-z0-9]+)"')
+_RE_EKN_URL_CAPTURE = re.compile(r'"ekn\:\/\/[a-z0-9-]*\/(?P<id>[a-z0-9]+)"')
 
```

Comment:
Tested this in Pythex and it works fine.

Only suggestion here would be to use `[a-z0-9\-_\.\\\/]` (that will catch ., _, \, / greedily as well, which is useful in case the leading path has multiple components).

Reply:
Thanks Sam, adjusted the PR.